constraintlayout/build-logic/src/main/groovy/androidx.build.base.gradle

```diff
@@ -8,7 +8,7 @@ version = providers.gradleProperty("${project.name}.version").forUseAtConfigurat
 pluginManager.withPlugin("java-base") {
     java {
         toolchain {
-            languageVersion.set(JavaLanguageVersion.of(8))
+            languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
I don't think this is enough. The project build.gradle file explicitly expects version 8.

Reply:
this is for the toolchain -- so you can get 11 but compile the code as 8